components/scream/src/share/tests/workspace_tests.cpp

```diff
@@ -168,6 +168,7 @@ static void unittest_workspace()
     }
 
 #ifndef KOKKOS_ENABLE_CUDA
+#ifdef WS_EXPENSIVE_TEST
```

Comment:
I'd like this to run at least once. Could you say that if nthreads is 2 (say), run this even w/o WS_EXPENSIVE_TEST defined?

Reply:
That's a good idea. Fix coming.